### DISCUSSION TITLE
Model for Vanadium Redox Flow Battery

### DISCUSSION THREAD

**Discussion by qazisajid2012** [pybamm-team/PyBaMM#3018] [General]
Hello All,

Its really an amazing platform to understand battery operation, performance and learn many things about batteries. But it would be great if model for VRFB can be included in PyBaMM then many new researchers like me will have good knowledge of VRFB. 

Or may be github is working on it already. If yes, then when it will be here for analysing?

Thanks

**Comment by brosaplanella** [comment]
There are no plans of including Vanadium Redox Flow Battteries, but depending on the equations needed you could implement the model yourself and use PyBaMM to solve it.
